zproject/local_settings_template.py

```diff
@@ -336,3 +336,12 @@
 
 # Controls whether Zulip will rate-limit user requests.
 # RATE_LIMITING = True
+
+# Full-text search configuration
+#
+# By default, Zulip supports full-text search only for English.  If
+# you want to use other languages, use PGroonga based full-text search
+# feature. See
+# https://zulip.readthedocs.org/en/latest/full-text-search.html how to
+# enable PGroonga based full-text search feature.
+# USING_PGROONGA = True
```

Comment:
maybe change this to just read `USING_PGROONGA = False` and then the user can change `False` -> `True` to enable it?  The instructions would be simpler this way.